### PR TITLE
AT_04.01_01 | Clickability of user name link

### PR DESCRIPTION
**Pull request by KristinaOzhog**
https://trello.com/c/ddrGDq3s

### CODE REVIEW

**Comment by DariaMas7** [comment]
Hello. The name of your commit should include AT name instead of TC
Also in description, you should attach a link to your AT card
The name of your branch should be initials of lastName firstName / name of the task
in your case for example ko/navigationbreadcrumbs

File: cypress/e2e/navigationWithBreadcrumbs.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="cypress"/>
+
+describe('navigationWithBreadcrumbs', () => {
+    it('AT_04.01_01 | Verify clickability of user name link in Breadcrumbs', () => {
+        cy.get('#side-panel div#tasks div:nth-child(5) a').click()
```
Comment (DariaMas7) [inline cypress/e2e/navigationWithBreadcrumbs.cy.js:5]: In your TC this line goes as a precondition so by this logic I would include this code in hooks or change the logic of TC

File: cypress/e2e/navigationWithBreadcrumbs.cy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+        cy.get('#breadcrumbBar li:nth-child(3)>a').click()
+        cy.get('#main-panel > div:nth-child(4)').contains('Jenkins User ID:')
```
Comment (DariaMas7) [inline cypress/e2e/navigationWithBreadcrumbs.cy.js:8]: Jenkins User ID should be in fixtures please

File: cypress/e2e/navigationWithBreadcrumbs.cy.js (hunk 1 above)
Comment (KristinaOzhog) [reply cypress/e2e/navigationWithBreadcrumbs.cy.js:7]: Thank you, done

File: cypress/e2e/navigationWithBreadcrumbs.cy.js (hunk 2 above)
Comment (KristinaOzhog) [reply cypress/e2e/navigationWithBreadcrumbs.cy.js]: Thank you, done

**Pull request closed without merging**